ds-collector/README.md

```diff
@@ -204,6 +204,16 @@ sed -i 's/^#?skipS3=.*/skipS3=\"true\"/'
 sed -i 's/^#?keepArtifact=.*/keepArtifact=\"true\"/'
 ```
 
+* If the script fails when attempting to run the `cqlsh` command due to an incorrect host address, an additional configuration item can be uncommented and used to supply a suitable host value.  The default behaviour of the setting is that a node will use the `hostname` command to provide the address to use as it is run on the node. Problems occur if this hostname does not resolve to the IP address that the node is listening for connections on. An alternative command can be used instead, which is supplied via configuration.
```

Comment:
we need to make mention that it is recommended and normal to first configure `cqlHost`.
that this is only needed when the hostname cannot be a standard ip/hostname on all nodes, and not `$(hostname)`

Reply:
I still curious if this comment/info is better here or here: https://github.com/datastax/diagnostic-collection/blob/master/ds-collector/collector.conf.in#L139